src/api/lib/tasks/dev/notifications.rake

```diff
@@ -21,7 +19,8 @@ namespace :dev do
       requestor_project = Project.find_by(name: 'requestor_project') || RakeSupport.create_and_assign_project('requestor_project', requestor)
 
       # Create notification for roles revoked
-      User.find_by(login: 'Iggy').run_as do
+      iggy = User.where(login: 'Iggy').first || create(:confirmed_user, login: 'Iggy', create_home_project: true)
```

Comment:
```suggestion
      iggy = User.find_by(login: 'Iggy') || create(:confirmed_user, login: 'Iggy', :with_home)
```
Using prevents from `find_by` query with `where` and then apply `first`.

Also, make use of the ` with_home` defined trait.

Reply:
done